app.rb

```diff
@@ -76,7 +76,7 @@ def perform(country_slug, legislature_slug, source = nil)
     end
     # Only use last 64k of output
     output = output[-64_000..-1] || output
-    title = "#{country.name} (#{legislature.name}): refresh data"
+    title = "#{country.name} (#{legislature.name}#{source ? ":#{source}" : ''}): refresh data"
     body = "Automated data refresh for #{country.name} - #{legislature.name}" \
```

Comment:
I think we’ll want to show this in the body text immediately below as well, so it’s probably better to pull the country/legislature/source combo into a variable and then interpolate it into both title and body (I don't think it matters that it's a slight change to the format we use in the body)